TSC-1/README.md

```diff
@@ -64,6 +64,18 @@ The typical process is:
 - The TSC discusses and refines specifications during its meetings
 - Accepted specifications are placed on the [roadmap](https://github.com/opendatakit/roadmap/projects/1)
 
+## Requesting feedback
```

Comment:
Can we please call this `Lazy consensus guidelines`

The title 'Requesting feedback' implies the audience of this document is committers/community members and not TSC members, which is inconsistent with other sections I believe. It also does nothing to link this section to the "establish appropriate guidelines" section in the TSC-GOVERNANCE.